probe-rs/src/debug/language.rs

```diff
@@ -61,7 +54,19 @@ pub trait ProgrammingLanguage {
     }
 }
 
-#[derive(Clone)]
-pub struct UnknownLanguage;
+#[derive(Debug, Clone)]
```

Comment:
This `Debug` doesn't really make sense. Everything is happening through a trait object reference, which isn't `Debug`.

Reply:
I prefer to have `Debug` implementations on everything, but I can remove it.